command/sub8_mission_control/sub8_mission_control/goal_server.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import rospy
+import actionlib
+import roslib; roslib.load_manifest('sub8_mission_control')
```

Comment:
Use setup.py to create a python module instead of roslib (I think you already did this - check if you actually need this line?)